api/ocm/cpi/repocpi/bridge_r.go

```diff
@@ -34,6 +34,16 @@ type RepositoryImpl interface {
 	io.Closer
 }
 
+type Chunked interface {
```

Comment:
Although the purpose of this interface is quite obvious in the context of this PR, it might not be without it. Please add a brief comment.

Reply:
done